_episodes/08-simulations.md

```diff
@@ -29,7 +29,7 @@ keypoints:
 > 
 > The PhD students have to think about the **numerical implementation** of their solution in the model.
 > 
-> PhD students will also work in pairs and will use the **same model version** (CESM1.2.2 with CAM 5.3) but can set-up a more appropriate compset if needed.
+> PhD students will also work in pairs and will use the **same model version** (CESM1.2.2 with CAM 5.3) but can set up a more appropriate compensate if needed.
```

Comment:
Heads up, _compset_ is the correct word here. It's short for component set (http://www.cesm.ucar.edu/models/cesm2/config/compsets.html).

Reply:
Get it. Maybe it's better to use the long name or maybe add (component set) here